README.md

```diff
@@ -1,2 +1,36 @@
 # renovate-config
+
 A sharable config preset for renovate used by SMG automotive.
+
+## Setup
+
+Enable Renovate in your repository and create a `renovate.json` file in the root of your project:
+
+````json
+{
+  "extends": [
+    "github>smg-automotive/renovate-config"
+  ]
+}
+````
+
+By default, renovate will pin all **dev** dependencies and uses SemVer ranges for the others. If you work on a project that
+is **not** `required()` by any other package, add `:pinDependencies` to renovate.json.
+
+````json
+{
+  "extends": [
+    "github>smg-automotive/renovate-config",
+    ":pinDependencies"
```

Comment:
alternatively, we could shoot for two separate, self-descriptive configs?
```
    "github>smg-automotive/renovate-config/node",
    "github>smg-automotive/renovate-config/pkg",
```

or similar - hope you get the idea (I don't recall what the differentiating factor is). I'm not sure this is even possible :D

Reply:
I don't recall there to be any significant difference other than not being strict about semantic commits in the web projects

Reply:
only difference is that for pkgs we want to only pin dev dependencies and use SemVer ranges for others. Whereas web projects would pin all the dependencies. Additionally the semantic commit flag that you mentioned in your comment above